stable/jenkins/ci/xml-values.yaml

```diff
@@ -0,0 +1,10 @@
+master:
+  enableXmlConfig: true
+  runAsUser: 0
+  fsGroup: 0
```

Comment:
I'm thinking the XML test shouldn't need to run as root

Reply:
root was once the default this just checks if it still works.